Makefile

```diff
@@ -0,0 +1,21 @@
+MANAGER_IMAGE ?= kubernetes-nmstate-manager
+
+all:
+	@echo Hello
+
+check: check-gofmt check-vet
+
+check-gofmt:
```

Comment:
From operator-sdk Makefile (is bigger project with more source code):
```make
format:
	go fmt $(PKGS)
```

They don't have new script just for this also fmt is fast enough so let's not check the stuff not formatted.